test/e2e/scheduling/limit_range.go

```diff
@@ -72,10 +72,6 @@ var _ = SIGDescribe("LimitRange", func() {
 		limitRanges, err := f.ClientSet.CoreV1().LimitRanges(f.Namespace.Name).List(context.TODO(), options)
 		framework.ExpectNoError(err, "failed to query for limitRanges")
 		framework.ExpectEqual(len(limitRanges.Items), 0)
-		options = metav1.ListOptions{
```

Comment:
👍🏾

Reply:
same comment about a buggy test, racing the create and the informer start

Reply:
Found a way to keep this as is and keep linter happy. Awaiting comments.

Reply:
same comment here, I think this is currently a race in the informer setup; we should open an issue for scheduling and leave this failing the check